handbook/engineering/distribution/managed/creation_process.md

```diff
@@ -2,8 +2,7 @@
 
 Creating a new managed instance involves following the steps below.
 
-1. Ask @stephen or @beyang to create a new GCP project `sourcegraph-managed-$COMPANY` and grant you IAM **Editor** role access.
-1. Ask @beyang to enable billing in the GCP project.
+1. Create a new GCP project `sourcegraph-managed-$COMPANY` under the "Managed instances" folder in the Sourcegraph GCP organization and grant yourself IAM **Editor** role access.
```

Comment:
```suggestion
1. Create a new GCP project `sourcegraph-managed-$COMPANY` under the "Managed instances" folder in the Sourcegraph GCP organization.
```

Permissions are not required, as the creator you get automatic "Owner" and all projects inside "Managed Instances" grant all members of distribution "Owner" as well.